src/components/BookList.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+const BooksList = ({ books }) => (
+
+  <table>
+    <thead>
+      <tr>
+        <th>Book Id</th>
+        <th>Title</th>
+        <th>Category</th>
+      </tr>
+      <tr>
```

Comment:
- [ ] _[REQUIRED]_ On running this project, I get a `TypeError: Cannot read property 'map' of undefined`. Kindly fix this.